tests/test_rule_in_container.sh

```diff
@@ -53,17 +55,21 @@ _arg_name="ssg_test_suite"
 _arg_scenarios=".*"
 _arg_datastream=
 _arg_remediate_using="oscap"
+_arg_dontclean="off"
+_arg_dry_run="off"
 
 
 print_help()
 {
 	printf '%s\n' "Test a rule using the container backend."
-	printf 'Usage: %s [-n|--name <arg>] [-s|--scenarios <arg>] [-d|--datastream <arg>] [-r|--remediate-using <REMEDIATION>] [-h|--help] <rule>\n' "$0"
+	printf 'Usage: %s [-n|--name <arg>] [-s|--scenarios <arg>] [-d|--datastream <arg>] [-r|--remediate-using <REMEDIATION>] [--(no-)dontclean] [--(no-)dry-run] [-h|--help] <rule>\n' "$0"
 	printf '\t%s\n' "<rule>: The short rule ID. Wildcards are supported."
 	printf '\t%s\n' "-n, --name: Name of the test image (default: 'ssg_test_suite')"
 	printf '\t%s\n' "-s, --scenarios: Regex to reduce selection of tested scenarios (default: '.*')"
 	printf '\t%s\n' "-d, --datastream: Path to the datastream to use in tests. Autodetected by default. (no default)"
 	printf '\t%s\n' "-r, --remediate-using: What to remediate with. Can be one of: 'oscap', 'bash' and 'ansible' (default: 'oscap')"
+	printf '\t%s\n' "--dontclean, --no-dontclean: Dont remove HTML reports from the log directory. (off by default)"
+	printf '\t%s\n' "--dry-run, --no-dry-run: Just print the test suite command-line. (off by default)"
```

Comment:
This is similar but not so terrible. However, I really don't understand the purpose of `--no-dry-run`. If I don't want to run a dry run I simply don't type `--dry-run`.

Reply:
There is a theoretical possibility that you have command options stored in an array, but you want to override any prior `--dry-run` that may or may not be there, and you would do that by adding `--no-dry-run` to the array. But again, I agree that it is unfortunate.